WTWSMS/common/landed_titles/EBS_titles.txt

```diff
@@ -0,0 +1,27 @@
+d_odoacer = { # Used for Odoacer's revolt
+	color = { 51 102 51 }
+	color2 = { 255 255 255 }
+
+	# Always exists
+	landless = yes
+	
+	allow = {
+		always = no
+	}
+	
+	culture = skirian
+	religion = arian
+
+	title = REX
+	title_female = REGINA
+
+	short_name = yes
+	
+	# Cannot be held as a secondary title
+	primary = yes
+	
+	# Cannot be vassalized
+	independent = yes
+
+	name_tier = KING
+}
```

Comment:
I'm not sure we need this separate file could be added at the end of the WTWSMS one for similar titles

Reply:
I added it under k_italy.